app/models/concerns/trs/retryable_client.rb

```diff
@@ -0,0 +1,17 @@
+module TRS
+  module RetryableClient
+    extend ActiveSupport::Concern
+
+    included do
+      retry_on Errors::APIRequestError,
+               wait: ->(executions) { 2**executions },
+               attempts: 15
```

Comment:
I just picked something that seemed vaguely sensible. 

Open to other suggestions, and also happy to iterate later if we want to see how this goes ♻️